roles/apache/tasks/main.yml

```diff
@@ -81,6 +81,14 @@
     notify:
       - reload apache
 
+  - name: write config changes to ssl.conf
+    lineinfile: dest=/etc/apache2/mods-available/ssl.conf
+                regexp="^\s*{{ item.key }}"
+                line='\t{{ item.key }} {{ item.value }}'
+    with_dict: "{{ apache.ssl.settings }}"
```

Comment:
hrm I'm not sure this should work.  `apache.ssl.settings` is not a dict, it's a string w/ line breaks in it.  I think just removing the `with_items` line and using the `apache.ssl.settings` var directly in the `lineinfile` would be more straightforward.